.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,38 @@
+name: Chromatic
+# All pull requests, and
+# Workflow dispatch allows you to run this workflow manually from the Actions tab
+on:
+  pull_request:
+    paths:
+      - packages/libs/react-ui/**
+  push:
+    branches:
+      - main
+    paths:
+      - packages/libs/react-ui/**
+  workflow_dispatch:
+
+jobs:
+  chromatic:
+    name: Visual Regression Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history, needed to determine diffs.
```

Comment:
I think it's better to only fetch the `main` branch right? You can do something like this:

```
      - name: Fetch base branch
        if: github.ref_name != 'main'
        run: git fetch origin ${{ github.base_ref }}
```

Then the `fetch-depth` can be set to `1` or `2`.

Reply:
Chromatic requires git history for all branches to be able to determine diffs ([according to the docs](https://www.chromatic.com/docs/github-actions#support-for-codeactionscheckoutv2code-and-above)) I'd be willing to try your suggestion but I'm not entirely sure it would work.

Reply:
Based on a different fetch depth, I had odd results with when it was able to determine diffs or not.